server/src/ws/dispatch.rs

```diff
@@ -0,0 +1,71 @@
+// SPDX-FileCopyrightText: 2023 Phoenix R&D GmbH <hello@phnx.im>
```

Comment:
It looks like these files have been moved? Shouldn't we move them to something thatisn't called "ws" instead, or is the final removal of the websocket in a different PR?